src/test/java/com/google/opensesame/MentorTest.java

```diff
@@ -1,32 +1,32 @@
-package com.google.opensesame;
+// package com.google.opensesame;
 
-import com.google.opensesame.servlets.MentorBuilder;
-import com.google.opensesame.servlets.MentorObject;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
+// import com.google.opensesame.servlets.UserBuilder;
+// import com.google.opensesame.servlets.UserObject;
+// import java.io.IOException;
+// import java.util.ArrayList;
+// import java.util.List;
+// import org.junit.Assert;
+// import org.junit.Test;
+// import org.junit.runner.RunWith;
+// import org.junit.runners.JUnit4;
 
-@RunWith(JUnit4.class)
-public class MentorTest {
-  private static final String MENTOR_NAME = "Mentor";
-  private static final String MENTOR_GITHUB_ID = "Obinnabii";
-  private static final String PROJECT_ID = "Project_ID";
+// @RunWith(JUnit4.class)
+// public class MentorTest {
+//   private static final String MENTOR_NAME = "Mentor";
+//   private static final String MENTOR_GITHUB_ID = "Obinnabii";
+//   private static final String PROJECT_ID = "OpenSesame";
```

Comment:
Let's remove commented out code rather than keeping it around.

Reply:
Agreed. Let's remove this and create new tests for the `UserData` and `UserEntity` classes.